plugins/netmotion.yaml

```diff
@@ -0,0 +1,106 @@
+# Plugin Info
+version: 0.0.1
+title: Netmotion
```

Comment:
Should include min stanza version that has the syslog enhancement for escape handler.